lib/podlet.js

```diff
@@ -288,49 +290,61 @@ const PodiumPodlet = class PodiumPodlet {
     }
 
     async process(incoming) {
-        incoming.view = this._view;
-        incoming.name = this.name;
-        incoming.css = this.cssRoute;
-        incoming.js = this.jsRoute;
+        const s = incoming;
```

Comment:
I am struggeling to understand the no re-assignment ESLint rule here. Having to assign the object to a new variable to alter the objects propertis have as I see it no value. Should we ignore this rule?

Reply:
yeah, agreed

Reply:
done